substrate/frame/vesting/src/benchmarking.rs

```diff
@@ -383,6 +383,34 @@ benchmarks! {
 		);
 	}
 
+force_remove_vesting_schedule {
+		let l in 0 .. MaxLocksOf::<T>::get() - 1;
+		let s in 2 .. T::MAX_VESTING_SCHEDULES - 1;
```

Comment:
I don't see why we couldn't have `MAX_VESTING_SCHEDULES` schedules since we just remove one.

```suggestion
		let s in 2 .. T::MAX_VESTING_SCHEDULES;
```

Reply:
Yeah we don't need this, it was added as a cautionary to not hit the upper bound when I was debugging last night.